_includes/tutorials/fk-joins/kstreams/markup/dev/start-compose.adoc

```diff
@@ -0,0 +1,5 @@
+And launch it by running:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/filtering/kstreams/code/tutorial-steps/dev/docker-compose-up.sh %}</code></pre>
```

Comment:
copy and paste moster...  `filtering` should be `fk-joins`

Reply:
ack, thanks for that.